source/element-kinds.md

```diff
@@ -5,8 +5,8 @@ Note: `name`, `length`, and `s_position` parameters stand alone and not part of
 
 Example:
 ```{code} yaml
-Solenoid:
-  name: cleo
+cleo:             # [string] user-defined name
+  kind: Solenoid  # [string] element kind
```

Comment:
The type of `kind` is better described as a `switch` since `string` implies that any set of characters is acceptable.